src/renderer/src/routes/pages/VoteNeo3/VoteNeo3ListItem.tsx

```diff
@@ -0,0 +1,197 @@
+import { cloneElement, useMemo, useRef } from 'react'
+import { useTranslation } from 'react-i18next'
+import { MdCircle, MdInfoOutline } from 'react-icons/md'
+import { TbPackages } from 'react-icons/tb'
+import { GetVoteDetailsByAddressResponse } from '@cityofzion/bs-neo3/dist/interfaces'
+import { Button } from '@renderer/components/Button'
+import { DashedLine } from '@renderer/components/DashedLine'
+import { IconButton } from '@renderer/components/IconButton'
+import { Tooltip } from '@renderer/components/Tooltip'
+import { VOTE_NEO3_COZ_PUB_KEY } from '@renderer/constants/public-keys'
+import { NumberHelper } from '@renderer/helpers/NumberHelper'
+import { StringHelper } from '@renderer/helpers/StringHelper'
+import { useMount } from '@renderer/hooks/useMount'
+import { TVoteNeo3Candidate } from '@shared/@types/query'
+import { IAccountState } from '@shared/@types/store'
+import { UseQueryResult } from '@tanstack/react-query'
+
+const columnColors = [
+  '76, 255, 179',
+  '0, 221, 180',
+  '46, 190, 129',
+  '71, 190, 255',
+  '125, 75, 147',
+  '151, 71, 255',
+  '211, 85, 231',
+  '231, 85, 149',
+  '254, 135, 47',
+  '254, 196, 47',
+  '188, 255, 45',
+]
```

Comment:
Aren't this color in tailwind theme ?

Reply:
The most of them do, but I'll add all the colors and use the opacity in another way.